docs/examples.rst

```diff
@@ -665,4 +665,15 @@ Finally, you can exclude single attributes from certain methods:
    ...     user = attr.ib()
    ...     password = attr.ib(repr=False)
    >>> C("me", "s3kr3t")
-   C(user='me')
+   C(user='me', password=***)
```

Comment:
(This presumably is @hynek's comment too, this one is not showing exclusion anymore now.)

Reply:
fixed in f056bce46bb7a6fcb68287b751bf15f88dd05e8d